### PR TITLE
Test compwiz

### PR DESCRIPTION
**Pull request by jjackson-s**
Began to write test for ComponentWizard().

### CODE REVIEW

**Comment by jjackson-s** [comment]
I tried to rebase this against my local version of the branch that you approved to merge (PR 75), but only the commits that I made after I checked out out the branch for this PR popped up. Do I need to wait until the previous PR is merged to be able to rebase it against the cfgwiz2?

**Comment by pmcollins** [comment]
I just merged your previous change. Can you try rebasing again?

File: cmd/configschema/configwiz/configwiz/component_test.go [hunk 1]
```diff
@@ -81,6 +83,65 @@ func buildTestCFGFields(name string, typ string, kind string, defaultStr string,
...
+	)
+	var fields []*configschema.Field
+	fields = append(fields, &newField)
+	cfgField.Fields = fields
+	componentWizard(io, 0, &cfgField)
+	return cfgField
+}
+
+func TestComponentWizard(t *testing.T) {
+	//if field.name == squash
```
Comment (pmcollins) [inline cmd/configschema/configwiz/configwiz/component_test.go:110]: Can we delete these comments?

File: cmd/configschema/configwiz/configwiz/component_test.go [hunk 2]
```diff
@@ -81,6 +83,65 @@ func buildTestCFGFields(name string, typ string, kind string, defaultStr string,
...
+	writerSquash := fakeWriter{}
```
Comment (pmcollins) [inline cmd/configschema/configwiz/configwiz/component_test.go:111]: Looks like this test function is doing four tests, right? Might be better to split it into four different tests, each with a meaningful name. Will that work?

Comment (jjackson-s) [reply cmd/configschema/configwiz/configwiz/component_test.go:111]: Gotcha, will split them up

Comment (pmcollins) [reply cmd/configschema/configwiz/configwiz/component_test.go:110]: Thanks!

**Review by pmcollins — APPROVED** [review]
LGTM

**Review by pmcollins — APPROVED** [review]
LGTM

**Pull request merged**